main.js

```diff
@@ -393,15 +393,16 @@ $(function() {
       return;
     }
 
-    //if player selected "buy all, count is set to highest purchesable amount
     const itemsAvailable = towns[currentTown].stockpiles[productIndex];
     const buttonText = $(this).text(); 
-    const count = ( buttonText  !== 'all') ? 1
+
+    //if player selected "buy all, count is set to highest purchesable amount
```

Comment:
This comment was not amended, it has some issues:

- no space in beginning
- no closing `"` sign after `all`
- purch*a*sable is misspelled

Correctly it is:
 ```
// if player selected "buy all, count is set to highest purchesable amount
```